src/inmanta/module.py

```diff
@@ -579,7 +579,8 @@ def to_config(self, inp: Optional[configparser.ConfigParser] = None) -> configpa
             out.add_section("metadata")
 
         for k, v in self.dict().items():
-            out.set("metadata", k, str(v))
+            if v:
```

Comment:
```suggestion
            if v is not None:
```
We still want to write booleans.

Reply:
but not empty strings?

Reply:
I'm not sure. I'd say an empty string is a value as well while `None` indicates the absence of a value. I'd write empty strings, provided the cfg format supports it.

Reply:
It seems like it does. I'm not getting any errors when running `pip install` against a file with `my_field =` lines.

Reply:
`configparser` parses it correctly too.

Reply:
agreed!